### PR TITLE
Introduction

### PR DESCRIPTION
**Pull request by chenhoward**

### CODE REVIEW

**Comment by negativetwelve** [comment]
I have a rake task that generates all files. It generates a folder called `_deploy` which is git ignored. You should either use this rake task or gitignore the `_site` directory.

File: chapters/introduction/index.md [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+### 2. View
+What the user interacts with.  Consists of the HTML, CSS, and javascript to
+display the data.
+
+### 3. Controller
+The connection between the models and the views.  It usually takes the model's
+data and populates the views with it.
+
+### 4. MVC Image
+![mvc-image](http://railstutorial.org/images/figures/mvc_detailed-full.png)
```
Comment (negativetwelve) [inline chapters/introduction/index.md]: we should be adding these images to our own repo

File: chapters/introduction/index.md [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+You can get a webapp up really fast, there is a large community, and it is open
+source. A lot of startups use Rails for these reasons.
+
+### About this Book
+
+In this book we are going to use Git for version control, Ruby version 2.2.0 and
+Rails 4.1.5.
+
+## Mac and Linux Installation
+
```
Comment (negativetwelve) [inline chapters/introduction/index.md:51]: Add a TODO for later?

File: chapters/introduction/index.md [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+### Before
+
+If you have Linux run in your console
+
+{% highlight console %}
```
Comment (negativetwelve) [inline chapters/introduction/index.md:56]: I'm pretty sure in markdown you can just indent 4 spaces and it'll infer the language for code blocks. Prevents having to wrap everything in highlight blocks

File: index.slim [hunk 4]
```diff
@@ -8,3 +11,6 @@ p
 p
   | By: 
   a href="http://markmiyashita.com" Mark Miyashita
+  |  and
```
Comment (negativetwelve) [inline index.slim]: use `'` and then you don't have to add trailing spaces

File: chapters/introduction/index.md (hunk 2 above)
Comment (chenhoward) [inline chapters/introduction/index.md:51]: A Todo for what?

File: chapters/introduction/index.md (hunk 3 above)
Comment (chenhoward) [inline chapters/introduction/index.md:56]: It does do a styling but I like the highlight console one better

**Comment by chenhoward** [comment]
Look at me again doe @negativetwelve

File: chapters/introduction/index.md (hunk 2 above)
Comment (negativetwelve) [inline chapters/introduction/index.md:51]: nevermind, didn't realize this was a section all on its own

File: chapters/introduction/index.md (hunk 3 above)
Comment (negativetwelve) [inline chapters/introduction/index.md:56]: cool

**Comment by negativetwelve** [comment]
git rm cached the _site directory please

**Comment by chenhoward** [comment]
Got it cooaaaaaaccch

**Pull request merged**